docs: k8s-jobs peonMonitors info extended (#14637)
apache/druid: docs/development/extensions-contrib/k8s-jobs.md

|`druid.indexer.runner.javaOptsArray`|`JsonArray`|java opts for the task.|`-Xmx1g`|No|
|`druid.indexer.runner.labels`|`JsonObject`|Additional labels you want to add to peon pod|`{}`|No|
|`druid.indexer.runner.annotations`|`JsonObject`|Additional annotations you want to add to peon pod|`{}`|No|

> |`druid.indexer.runner.peonMonitors`|`JsonArray`|Overrides peon pod's `druid_monitoring_monitors` environment variable. Use this property if you don't want to inherit monitors from the Overlord.|`[]`|No|

Reviewer: ```suggestion
|`druid.indexer.runner.peonMonitors`|`JsonArray`|Overrides the `druid.monitoring.monitors` environment variable of a peon pod. Use this property if you don't want to inherit monitors from the Overlord.|`[]`|No|
```

configure monitors

  name: druid-cluster
  apiGroup: rbac.authorization.k8s.io
```

> To use `druid.indexer.runner.peonMonitors` option properly define Overlord's monitors like below:

Reviewer: ```suggestion
To use the `druid.indexer.runner.peonMonitors` option, configure monitors on the Overlord as follows:
```
